components/home/__tests__/homepage-whatsapp-cta.test.tsx

```diff
@@ -44,4 +45,12 @@ describe("HomePageContributing", () => {
       </section>
     `);
   });
+
+  it("should navigate to ", () => {
+    const { getByText } = render(<HomePageWhatsAppCTA />);
+
+    const link = getByText("Hubungi hotline sekarang").closest("a");
+
+    expect(link).toHaveAttribute("href", siteConfig.whatsapp_contact_url);
```

Comment:
In my perspective, I never use this kind of dynamic assertion.
I prefer to use a static assertion.

In this case 

```js
expect(link).toHaveAttribute("href", "bit.ly/something");
```

Why? Because we want the test is capture the changes from the dynamic variable.
So in the future, if the value of `whatsapp_contact_url` somehow changed, then the test will failed.
And we will know that something impacted from our changes.

But, just my two cents 😂

Reply:
Ahh, that makes sense.

My thought was, it will make sure that the URL is the same as the config stated.

Got it! will waiting for further notes to take action

Reply:
In this case, we need to use the dynamic value, Mas @mazipan.
The reason is, `siteConfig` is managed by Netlify CMS, so when they make changes (and merge it to the `main` branch through Netlify CMS), it will break all the open PRs. 😅 

I agree with your point, but it's only applicable for values that we completely control from our code side without any other intervention such as Netlify CMS and Google Sheets edits.